en/docs/deploy-and-publish/publish-on-dev-portal/third-party-api-support.md

```diff
@@ -36,3 +36,9 @@ You can see an indicator added to the API thumbnail in the API Publisher Portal
 <a href="{{base_path}}/assets/img/develop/third-party-tag.png"><img src="{{base_path}}/assets/img/develop/third-party-tag.png" width="20%"></a>
 
 You can use the new default property `thirdParty` to search for APIs that are from a third-party.
+
+!!! note
+    Third-party API (REST API) resources can be used in API products. External Production/Sandbox endpoints of the APIs will be added as the endpoints for the API product. Endpoint security is not added for third-party API resources.
+
+!!! note
+    Subscriptions are available by default for third party APIs. But since the third APIs are not deployed in the API gateway, the tokens generated from API Manager applications will not be valid. In order to support tokens and other functionalities from API Manager for third party APIs, a special setup is required with the credentials of the service providers available within API Manager.
```

Comment:
```suggestion
    Subscriptions are available by default for third-party APIs. However, since the third-party APIs are not deployed in the API Gateway, the tokens generated from API Manager applications will not be valid. In order to support tokens and other functionalities from API Manager for third-party APIs, a special setup is required with the credentials of the service providers available within API Manager.
```